llama-index-integrations/llms/llama-index-llms-nvidia/tests/test_text-completion.py

```diff
@@ -52,9 +52,8 @@ async def mock_async_completion_v1(*args: Any, **kwargs: Any) -> Completion:
     return mock_completion_v1(*args, **kwargs)
 
 
-def mock_chat_completion_stream_v1(
-    model_name: str,
-) -> Generator[Completion, None, None]:
+def mock_completion_stream_v1(*args, **kwargs) -> Generator[Completion, None, None]:
+    model_name = kwargs.get("model")
```

Comment:
the model name doesn't appear to be important here. can you simply make it a static string?